README.md

```diff
@@ -116,28 +129,31 @@ Plotting graphs works the same way for both Spec2Vec and MS2DeepScore.
 
 The following example will plot the structures of the molecules
 ```python
-from omigami import MoleculePlotter
+from omigami.plotting import MoleculePlotter
 plotter = MoleculePlotter()
-plotter.plot_molecule_structure_grid(best_matches, 
-                                     draw_indices=True, 
-                                     molecule_image_size=[600, 600], 
-                                     substructure_highlight="C(=O)")
+plotter.plot_molecule_structure_grid(
+    spectra_matches=best_matches,
+    representation="smiles",
+    draw_indices=True,
+    img_size=(600, 600),
+    substructure_highlight="C(=O)"
+)
```

Comment:
not sure this is correct, we are returning a tuple from this now, but I can change this in the mirror plot PR

Reply:
which part is not correct? can you elaborate please?

Reply:
O.o I see know, I will correct it thanks for catching this!